pkg/resource/testdata/TestStatefulSetBuilder/default_insecure_statefulset.golden

```diff
@@ -77,14 +77,33 @@ spec:
         volumeMounts:
         - mountPath: /cockroach/cockroach-data/
           name: datadir
+      securityContext:
+        fsGroup: 10001
+        runAsUser: 10001
       serviceAccountName: cockroach-database-sa
       terminationGracePeriodSeconds: 60
       volumes:
       - name: datadir
         persistentVolumeClaim:
           claimName: ""
+      initContainers:
+        - command:
+          - /bin/sh
+          - -c
+          - '>- cp -p /cockroach/cockroach-certs-prestage/..data/* /cockroach/cockroach-certs/
+            && chmod 700 /cockroach/cockroach-certs/*.key && chown 10001:10001 /cockroach/cockroach-certs/*.key'
+          image: registry.access.redhat.com/ubi8/ubi
```

Comment:
I think if we're using the ubi image we're going to have to actually add that as a RELATED_IMAGE to the CSV, which we don't want to do. Let's either use the operator image or the db image to take this action.